code/game/machinery/computer/arcade.dm

```diff
@@ -15,16 +15,39 @@
 	var/enemy_mp = 25
 	var/gameover = 0
 	var/blocked = 0 //Player cannot attack/heal while set
-	var/list/prizes = list( /obj/item/storage/box/MRE = 3,
-							/obj/item/spacecash/c10 = 4,
-							/obj/item/ammo_magazine/flamer_tank = 1,
-							/obj/item/tool/lighter/zippo = 2,
-							/obj/item/tool/weldingtool = 1,
-							/obj/item/storage/box/uscm_mre = 2,
-							/obj/item/device/camera = 2,
-							/obj/item/device/camera_film = 4,
-							/obj/item/cell/crap/empty = 3,
-							/obj/item/tool/hand_labeler = 1
+	var/list/prizes = list( /obj/item/tool/lighter/zippo = 4,
+							/obj/item/facepaint/sniper = 4,
+							/obj/item/toy/gun = 4,
+							/obj/item/toy/crossbow = 4,
+							/obj/item/toy/sword = 5,
+							/obj/item/toy/katana = 1,
+							/obj/item/toy/spinningtoy = 2,
+							/obj/item/toy/prize/ripley = 2,
+							/obj/item/toy/prize/fireripley = 2,
+							/obj/item/toy/prize/deathripley = 2,
+							/obj/item/toy/prize/gygax = 2,
+							/obj/item/toy/prize/durand = 2,
+							/obj/item/toy/prize/honk = 2,
+							/obj/item/toy/prize/marauder = 2,
+							/obj/item/toy/prize/seraph = 2,
+							/obj/item/toy/prize/mauler = 2,
+							/obj/item/toy/prize/odysseus = 2,
+							/obj/item/toy/bikehorn = 2, // honk
+							/obj/item/clothing/head/collectable/tophat/super = 2,
+							/obj/item/toy/plush/farwa = 2,
+							/obj/item/toy/plush/barricade = 2,
+							/obj/item/toy/plush/shark = 2,
+							/obj/item/toy/plush/bee = 2,
+							/obj/item/toy/plush/moth = 2,
+							/obj/item/toy/plush/rock = 5,
+							/obj/item/storage/box/snappops = 5,
+							/obj/item/facepaint/lipstick = 1,
+							/obj/item/facepaint/lipstick/jade = 1,
+							/obj/item/clothing/head/headband/gray = 5,
+							/obj/item/clothing/head/headband/red = 5,
+							/obj/item/clothing/mask/tornscarf = 5,
+							/obj/item/clothing/shoes/stompers = 1
```

Comment:
I'll make this more clear, as a person who added them in the first place, i don't think they fit in this list logically, remove them